libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt

```diff
@@ -83,21 +83,8 @@ fun ElementLogoAtom(
         )
         Image(
             modifier = Modifier
-                .size(size.logoSize)
```

Comment:
this is a weird shape that appears behind logo on onboarding screen. Good riddance!